### DISCUSSION TITLE
EasyImage2.0 简单图床 相关教程

### DISCUSSION THREAD

**Discussion by icret** [icret/EasyImages2.0#86] [公告]
EasyImage2.0 简单图床

- 使用手册
https://icret.gitee.io/easyimages2.0/#/

- 宝塔搭建教程
https://blog.laoda.de/archives/aapanel-install-easyimage

- docker 版
https://hub.docker.com/r/ddsderek/easyimage

- docker 版教程
https://blog.laoda.de/archives/docker-compose-install-easyimage

**Comment by wayner6** [comment]
根据我自己的使用，补充一点，宝塔面板请不要开启`防跨站攻击`。否则会提示 php 错误 open_basedir restriction in effect

**Reply by icret** [reply]
> 根据我自己的使用，补充一点，宝塔面板请不要开启`放跨站攻击`。否则会提示 php 错误 open_basedir restriction in effect

你补充的很对，而且教程已经过了。包括安装过程中也有提示！

**Reply by wayner6** [reply]
好的
